cluster/gce/gci/configure-helper.sh

```diff
@@ -453,7 +462,7 @@ function mount-master-pd {
   # locations.
 
   # Contains all the data stored in etcd.
-  mkdir -m 700 -p "${mount_point}/var/etcd"
+  install -d -m 700 "${mount_point}/var/etcd"
```

Comment:
why do we need this?  

if this in response to https://github.com/koalaman/shellcheck/wiki/SC2174 I'd rather be explicit about preserving the existing behavior instead of trying to "fix" it, unless you can point to bugs that are caused by this

Reply:
You are correct that it is indeed SC2174 that I intended to mitigate here.
If you prefer the 2-liner suggested in https://github.com/koalaman/shellcheck/wiki/SC2174, I will of course update to that instead. (I chose install since it is a common tool and could be used in a oneliner fix, no other reason.)